docs/reference_info_json.md

```diff
@@ -342,9 +342,21 @@ Configures the [LED Matrix](feature_led_matrix.md) feature.
                 * The key matrix position associated with the LED.
                 * Example: `[0, 2]`
             * Example: `{"matrix": [2, 1], "x": 20, "y": 48, "flags": 2}`
+    * `led_flush_limit`
+        * Limits in milliseconds how frequently an animation will update the LEDs.
+        * Default: `16`
+    * `led_process_limit`
+        * Limits the number of LEDs to process in an animation per task run (increases keyboard responsiveness).
+        * Default: `led_count / 5`
     * `max_brightness`
         * The maximum value which brightness is scaled to, from 0 to 255.
         * Default: `255`
+    * `react_on_keyup`
+        * Animations react to keyup instead of keydown.
```

Comment:
Pretty sure this does not conflict with the keypresses define, you can have both enabled.

Reply:
This one takes precedence, you can have both defined but it will only do one will take effect.

https://github.com/qmk/qmk_firmware/blob/4a51f06c830abf9f3d82a78b6a6eff64539e0e2b/quantum/rgb_matrix/rgb_matrix.c#L224-L228

With `RGB_MATRIX_KEYPRESSES` now set if animations are enabled, we only need to expose this single option.

Reply:
Was this changed recently? I'm not sure I like that...

Reply:
The code linked? its basically always been that way.